decoder/gym.go

```diff
@@ -374,11 +376,77 @@ func createGymFortWebhooks(oldGym *Gym, gym *Gym) {
 	}
 }
 
+func makeRaidWebhook(gym *Gym) (payload map[string]interface{}) {
```

Comment:
Even if that is not already optimized by the compiler already, a map allocation should still be virtually for free.

Reply:
I expect you're right, it was more a style discussion. It seems the go language recommendations are only to use it on short functions where you can see the definition and the return on a single page, and warning against re-using the variable because it can be shadowed.